run-pytype.py

```diff
@@ -107,34 +106,28 @@ def main(me: str, branch_url: str):
         output_file (str): output file path for the markdown summary table
         branch_url (str): _url of the branch for file links in the summary table
     """
-    never = (
-        "xcp/repository.py",
-        "tests/test_ifrename_logic.py",
-        "tests/test_xmlunwrap.py",
-    )
-    excludes = [
-        "xcp/cmd.py",
-        "xcp/net/ip.py",
-    ]
-    errors_in = excludes.copy()
-    errors_in.extend(never)
+    broken_files = []  # type: list[str]
+    xfail_files = []  # type: list[str]
+    errors_in = xfail_files.copy()
+    errors_in.extend(broken_files)
```

Comment:
Extending by known empty list (since it was defined as empty on 109)

Reply:
Fixed, these variables were used for configuration, now they are read from config files!
(instead of `broken_files`, we can use the `exclude` config setting in `pytype.cfg`)